composer.json

```diff
@@ -25,8 +25,8 @@
         }
     },
     "require-dev": {
-        "vimeo/psalm": "^4.0|^5",
-        "nikic/php-parser": "^4.0"
+        "vimeo/psalm": "dev-master",
```

Comment:
I don't think this change is on purpose?

Reply:
Well, it does not affect dependencies anyway, I made it to test that everything is scanned fine

Reply:
I may have made a mistake somewhere when creating this, but I'd expect some constraint to prevent a user from requiring this plugin with a Psalm in version 3 or in version 7 for example.

if require-dev is not checked, then maybe that constraint should be put in require?

Reply:
Sure!